securedrop/sass/modules/_confirm-modal.sass

```diff
@@ -23,25 +23,38 @@
     height: 100%
 
   div
-    width: 400px
+    left: 84px
+    width: 650px
     position: relative
-    margin: 10% auto
-    padding: 5px 20px 13px 20px
-    border-radius: 10px
+    margin: 20% auto
+    padding: 20px 20px 43px 40px
     background: white
+    -moz-box-shadow: 0px 0px 15px 0px rgba(0,0,0,0.5)
+    box-shadow: 0px 0px 15px 0px rgba(0,0,0,0.5)
+    border: 1.5px solid #000c6c
+
+    p,li
+      font-size: 0.9em
+      line-height: 1.4em
+      color: #303030
+
+    b
+      color: #000000
+
+    h2
+      font-size: 22px
 
   .close
-    background: $color_grey_medium
+
+    background: $color_button_blue
```

Comment:
I think here you may want `$color_deep_blue` [1]

[1] https://github.com/freedomofpress/securedrop/blob/develop/securedrop/sass/_variables.sass#L20

Reply:
done.